src/ray/raylet/worker.h

```diff
@@ -117,6 +117,9 @@ class WorkerInterface {
 
   virtual const ActorID &GetRootDetachedActorId() const = 0;
 
+  virtual void SetIdleKeepAliveDeadline(absl::Time deadline) = 0;
+  virtual bool IsIdleKillable(absl::Time now) const = 0;
+
```

Comment:
I feel we don't have to change `worker.h`. We can change `std::list<std::pair<std::shared_ptr<WorkerInterface>, int64_t>> idle_of_all_languages_;` to store the deadline instead of when the worker becomes idle?

Reply:
done